scout/parse/variant/callers.py

```diff
@@ -48,4 +48,10 @@ def parse_callers(variant, category="snv"):
             filter_status = "Filtered - {}".format(filter_status.replace(";", " - "))
         callers["gatk"] = filter_status
 
+    if category == "sv" and not raw_info or other_info:
+        filter_status = "Pass"
+        if variant.FILTER is not None:
+            filter_status = "Filtered - {}".format(filter_status.replace(";", " - "))
+        callers["manta"] = filter_status
```

Comment:
That's because if the variant is called then it means that has passed in manta, right?

Reply:
It still feels a bit arbitrary to me, especially thinking about other facilities that might have different pipelines..

Reply:
It is very arbitrary. I am not fond of it, but it follows from the same pattern we have for the SNVs with at least MIP<11. Lund use the more sensible caller tags so they should be fine. And for others, I don’t think seeing manta:pass is any worse than all callers:pass…